slasher/rpc/detect_update_min_max_span_bench_test.go

```diff
@@ -0,0 +1,81 @@
+package rpc
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/slasher/db"
+)
+
+func BenchmarkMinSpan(b *testing.B) {
+	diffs := []uint64{2, 10, 100, 1000, 10000, 53999}
+	dbs := db.SetupSlasherDB(b)
+	defer db.TeardownSlasherDB(b, dbs)
+
+	ctx := context.Background()
+	slasherServer := &Server{
+		SlasherDB: dbs,
+	}
+	for _, diff := range diffs {
+		b.Run(fmt.Sprintf("MinSpan_diff_%d", diff), func(ib *testing.B) {
+			for i := uint64(0); i < uint64(ib.N); i++ {
+				_, err := slasherServer.DetectAndUpdateMinEpochSpan(ctx, i, i+diff, i%10)
+				if err != nil {
+					b.Fatal(err)
+				}
+			}
+		})
+	}
+}
+
+func BenchmarkMaxSpan(b *testing.B) {
+	diffs := []uint64{2, 10, 100, 1000, 10000, 53999}
+	dbs := db.SetupSlasherDB(b)
+	defer db.TeardownSlasherDB(b, dbs)
+
+	ctx := context.Background()
+	slasherServer := &Server{
+		SlasherDB: dbs,
+	}
+	for _, diff := range diffs {
+		b.Run(fmt.Sprintf("MaxSpan_diff_%d", diff), func(ib *testing.B) {
+			for i := uint64(0); i < uint64(ib.N); i++ {
+				_, err := slasherServer.DetectAndUpdateMaxEpochSpan(ctx, diff, diff+i, i%10)
```

Comment:
whats the `i%10` for ?

Reply:
to use few validators indices and have multiple updates on each of the maps